src/action_types/ui_action_types.js

```diff
@@ -0,0 +1,3 @@
+export const SHOW_IMPORT_DIALOG = 'SHOW_IMPORT_DIALOG';
```

Comment:
it might make sense to use a folder structure more similar to kabisa apps we create, where we'd do something such as

```
src/
  state/
    ui/
      actions.js
      reducers.js
      constants.js
```

not saying this approach is wrong, but it can get out of hand quickly